app/controllers/api/feed_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Api
+  class FeedController < ApiBaseController
+    def index
+      user = User.find_by_auth_token(params[:auth_token])
```

Comment:
@maskedXchaotic we may need authtntication for other api endpoings  current_user_from_token